plugins/node/opentelemetry-instrumentation-express/src/instrumentation.ts

```diff
@@ -244,6 +245,19 @@ export class ExpressInstrumentation extends InstrumentationBase<
         const span = instrumentation.tracer.startSpan(spanName, {
           attributes: Object.assign(attributes, metadata.attributes),
         });
+
+        if (instrumentation.getConfig().requestHook) {
+          safeExecuteInTheMiddle(
+            () => instrumentation.getConfig()?.requestHook!(span, type, req),
```

Comment:
you have optional chaining after the call to `getConfig()` which is lacking in the same statement on the `if` above it. 
I think `getConfig` will not return falsy values, so no need for this `?`, but if we do use it let's stay consistent.